src/main/java/frc/robot/subsystems/Gripper/Gripper.java

```diff
@@ -0,0 +1,101 @@
+package frc.robot.subsystems.Gripper;
+
+import org.littletonrobotics.junction.Logger;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Robot;
+
+
+public class Gripper extends SubsystemBase{
+    private final GripperIO io;
+    private final GripperInputsAutoLogged inputs = new GripperInputsAutoLogged();
+    private boolean hasCoral;
+    private boolean hasAlgae;
+
+    public Gripper(){
+        if (Robot.isReal()){
+            io = new GripperIOReal();
+        } else {
+            io = new GripperIOSim();
+        }
+
+        hasCoral = GripperConstants.START_WITH_CORAL; // Change to be based on whether or not the robot is on field
```

Comment:
Everything that is a "TODO" should be marked as:
```suggestion
        hasCoral = GripperConstants.START_WITH_CORAL; // TODO(owner name): Change to be based on whether or not the robot is on field
```